src/Discord/Provider.php

```diff
@@ -61,6 +61,24 @@ protected function getUserByToken($token)
         return json_decode($response->getBody()->getContents(), true);
     }
 
+    /**
+     * @param array $user
+     *
+     * @return string|null
+     */
+    protected function formatAvatar(array $user)
+    {
+        if (is_null($user['avatar'])) {
+            return null;
+        }
+
+        $isGif = preg_match('/a_.+/m', $user['avatar']) === 1;
```

Comment:
Can you explain this please?

Reply:
As explained on the docs for discord animated/gif icon starts with a_

Reply:
Thanks, see comment below :)